device/bgp/bgpv4.yaml

```diff
@@ -14,6 +14,22 @@ components:
             IPv4 address of the BGP peer for the session.
           type: string
           format: ipv4
+        evpn_ethernet_segments:
```

Comment:
We have to keep the v4 and v6 ethernetsegments separate because of our requirement. I think its better if we can have some knob to control the ordering of the parameters.